app/src/components/Modals/add-user-modal.tsx

```diff
@@ -27,36 +27,37 @@ import { TFunction } from "i18next";
 interface AddUserModalProps {
   isOpen: boolean;
   onClose: () => void;
-  cityId: string | undefined;
   t: TFunction;
+  userInfo: UserAttributes;
+  defaultCityId: string | undefined;
```

Comment:
```suggestion
  defaultCityId?: string;
```
Works too and is more readable as an optional parameter imo